src/index.ts

```diff
@@ -34,35 +31,53 @@ export { default as LayoutLinelengthContainer } from './components/LayoutLinelen
 export { default as LayoutSection } from './components/LayoutSection';
 export { default as Link } from './components/Link';
 export { default as LinkList } from './components/LinkList';
-export { default as ListDetail } from './components/ListDetail';
-export { default as ListDetailPanel } from './components/ListDetailPanel';
-export { default as Logo } from './components/Logo';
-export { default as LogoImage } from './components/LogoImage';
-export { default as Main } from './components/Main';
-export { default as NavContainer } from './components/NavContainer';
-export { default as NotificationList } from './components/NotificationList';
-export { default as NotificationListItem } from './components/NotificationListItem';
 export { default as OverflowList } from './components/OverflowList';
 export { default as OverflowListItem } from './components/OverflowListItem';
 export { default as NumberIcon } from './components/NumberIcon';
 export { default as PageHeader } from './components/PageHeader';
 export { default as Panel } from './components/Panel';
-export { default as Popover } from './components/Popover';
-export { default as PopoverBody } from './components/PopoverBody';
-export { default as PopoverFooter } from './components/PopoverFooter';
-export { default as PopoverHeader } from './components/PopoverHeader';
-export { default as PrimaryNav } from './components/PrimaryNav';
-export { default as PrimaryNavItem } from './components/PrimaryNavItem';
 export { default as Section } from './components/Section';
-export { default as ShowHide } from './components/ShowHide';
 export { default as Tab } from './components/Tab';
 export { default as Tabs } from './components/Tabs';
 export { default as Tag } from './components/Tag';
 export { default as Text } from './components/Text';
 export { default as TextPassage } from './components/TextPassage';
 export { default as Toast } from './components/Toast';
 export { default as Tooltip } from './components/Tooltip';
-export { default as UtilityNav } from './components/UtilityNav';
-export { default as UtilityNavItem } from './components/UtilityNavItem';
-export * from './hooks';
```

Comment:
Is `./hooks` only used in Project Overview?

Reply:
it's only used in `UtilityNav`, I'll leave a comment